jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -0,0 +1,499 @@
+#!groovy?
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonOutput
+import groovy.transform.Field
+@Field def Util
+
+echo "Environment and deployment metadata module loaded successfully"
+
+/**
+ * The environment-deployment loader module. This module will load all environment and deployment related  metadata from respective  tables or Apis 
+ * get it ready for Jenkins builds. It loads data for all service types. 
+*/
+
+
+@Field def g_login_token
+@Field def service_config
+@Field def config_loader
+@Field def scm_module
+@Field def g_service_branch 
+@Field def g_git_commit_hash
+@Field def g_git_commit_url
+@Field def g_git_token
+@Field def g_build_url
+@Field def g_build_id
+@Field def g_environment_logical_id
+@Field def g_environment_endpoint
+@Field def g_request_id
+@Field def g_base_url
+
+@Field def GIT_REPO_BASE_URL= "https://bitbucket.corporate.t-mobile.com/scm/cas/"
+
+/**
+ * Initialize the module
+ */
+def initialize(service_config, config_loader, scm_module,branch, buildUrl, buildId,baseUrl) {
+	setServiceConfig(service_config)
+	setServiceConfig(scm_module)
+	setConfigLoader(config_loader)
+	setBranch(branch)
+	setBuildId(buildId)
+	setBuildUrl(buildUrl)
+	setBaseUrl(baseUrl)
+}
+
+
+def getRepoCommitHash() {
+	if(g_git_commit_hash == null) {
+		g_git_commit_hash = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+	}
+	echo "SCM_Commit_Hash:$g_git_commit_hash"
+	return g_git_commit_hash
+	
+}
+
+def getRepoURL() {
+	if(g_git_commit_url == null) {
+		def bitbucket_service_id
```

Comment:
rename this variable to service_id

Reply:
or better rename this repo_name as this is not service_id

Reply:
Renamed